apollo-compiler/src/main/kotlin/com/apollographql/apollo3/compiler/ir/IrBuilder.kt

```diff
@@ -584,10 +589,12 @@ internal fun GQLValue.toIrValue(): IrValue {
  * - False
  * - (!)Variable
  * - (!)Variable & (!)Variable
+ *
+ * TODO: rename to toIncludeBooleanExpression
```

Comment:
Let's do it? 

* `toBooleanExpression` looks at all directives (for fragments)
* `toIncludeBooleanExpression` only looks at `include` and `skip` directives (for fields where we know there is no defer)
* `toDeferBooleanExpression` only looks at `defer` directives (for balance ⚖️ )

Reply:
Woops forgot about it! Yes makes sense 👍